lab-lee/app/view/home/home.html

```diff
@@ -0,0 +1,32 @@
+<main class="home">
+
+  <create-gallery> </create-gallery>
+
+  <h2>Filter by gallery...</h2>
+  <div class="search-bar-container">
+    <p>Search by name</p>
+    <search-bar
+    class="name-search"
+    term="homeCtrl.name">
+    </search-bar>
+
+    <p>Search by description</p>
+    <search-bar
+    class="desc-search"
+    term="homeCtrl.desc">
```

Comment:
Nice work componentizing your search bar and passing appropriate data down through bindings :)